app-admin/newsyslog/newsyslog-1.1.0.81-r1.ebuild

```diff
@@ -11,22 +11,23 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~hppa ~mips ~ppc ~ppc64 ~sparc ~x86"
 
-DEPEND="
-	sys-apps/groff"
-
+DEPEND="sys-apps/groff"
 RDEPEND="
 	virtual/cron
 	app-arch/gzip"
 
-PATCHES=( "${FILESDIR}/${P}-html.patch" )
+PATCHES=(
+	"${FILESDIR}"/${P}-html.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
 
 DOCS=( newsyslog.conf AUTHORS ChangeLog INSTALL NEWS ToDo )
 
 src_configure() {
-	local myconf="--with-syslogd_pid=/var/run/syslog.pid"
-
 	if has_version 'app-admin/syslog-ng'; then
 		myconf="--with-syslogd_pid=/var/run/syslog-ng.pid"
+	else
+		myconf="--with-syslogd_pid=/var/run/syslog.pid"
 	fi
```

Comment:
can't you do this as a bash array instead?

Reply:
Sure